cclib/parser/molcasparser.py

```diff
@@ -421,26 +423,162 @@ def extract(self, inputfile, line):
 
                 line=next(inputfile)
 
-            self.set_attribute('temperature', temperature_values[-1])
+            index  = -1
```

Comment:
Could you add a comment around here about what we're doing with the temperatures? (as in, returning room temp or last one if not present)